src/README.md

```diff
@@ -31,11 +31,11 @@ As an air traffic controller
 So I can get passengers to a destination
 I want to instruct the airport to land a plane
 
-| Objects | Properties                 | Messages                   | Outputs |
-| ------- | -------------------------- | -------------------------- | ------- |
-| Plane   | tailNumber @string         | gettailNumber()            | @string |
-| Airport | airport @array[tailNumber] | addtailNumber(@tailNumber) | @void   |
-|         |                            |                            |         |
+| Objects | Properties                       | Messages              | Outputs  |
+| ------- | -------------------------------- | --------------------- | -------- |
+| Plane   | tailNumber @string               | gettailNumber(Plane)  | @string  |
+| Airport | slotsOccupied @array[tailNumber] | addPlane(@tailNumber) | @boolean |
+|         |                                  |                       |          |
```

Comment:
Was getting a little confused with my domain modelling and made some changes, still doing further reading on it.